src/commonmark/en/content/metadata/Cause_of_death.md

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
```

Comment:
This file seems to be in the correct place.
But it is not markdown, it is HTML.

Please convert to markdown.